OrbitClientModel/CaptureSerializer.cpp

```diff
@@ -25,6 +26,10 @@ using orbit_client_protos::FunctionInfo;
 using orbit_client_protos::FunctionStats;
 using orbit_client_protos::TimerInfo;
 
+namespace {
+inline const std::string kFileOrbitExtension = ".orbit";
```

Comment:
I think the recommended way for this would be:
inline constexpr std::string_view kFileOrbitExtension = ".orbit";

Reply:
Done!